src/lib/models/aic.rs

```diff
@@ -60,6 +61,14 @@ impl AICModel<'_> {
         )
         .fetch_one(self.db_pool)
         .await
+        .map_err(|e| {
+            tracing::error!(
+                r#type = "aic-record-create-failed",
```

Comment:
i'm not familiar with this `r#type` syntax

Reply:
From the actix mozlog docs: "Since type is a Rust reserved keyword, this can also be specified using a raw-string-inspired format: r#type = value."

It looks like the `r#` is actually escaping the keyword name. See this: https://www.reddit.com/r/rust/comments/bb3ysz/today_i_learned_you_can_use_language_keywords_as/